app/services/doc_auth/mock/result_response.rb

```diff
@@ -254,6 +255,14 @@ def create_response_info(
           extra: { liveness_checking_required: liveness_enabled },
         }.compact
       end
+
+      def failed_file_data(failed_data)
+        if attention_with_barcode?
+          failed_data = failed_data.dig('failed_alerts')
+          failed_data.delete(ATTENTION_WITH_BARCODE_ALERT)
+        end
+        failed_data
```

Comment:
Should we return failed_data.dig('failed_alert')

Reply:
The `ATTENTION_WITH_BARCODE_ALERT` seems the default alert if none specified. but here we completely removed it also?

Reply:
@dawei-nava  - default alert? pls expound 🤔

Reply:
@amirbey , Sorry i was referring to DEFAULT_FAILED_ALERTS, which was set when there is no alerts passed in yaml file.

:+1 on fixing the issue.